mobile/src/main/java/org/openhab/habdroid/model/Item.kt

```diff
@@ -338,6 +342,12 @@ fun JSONObject.toItem(): Item {
         emptyList()
     }
 
+    val linkToMore = try {
+        optJSONObject("metadata")?.optJSONObject("link_to_more")?.optStringOrNull("value")?.toUri()
```

Comment:
Not sure if this should be made a well-known metadata or keep it as a custom one.